src/lambda/lambdaTreeDataProvider.ts

```diff
@@ -111,12 +127,20 @@ export class LambdaTreeDataProvider implements vscode.TreeDataProvider<AWSTreeNo
                 element: node,
                 outputChannel: this.lambdaOutputChannel,
                 resourceFetcher: this.resourceFetcher
-            })
+            }),
+            telemetryName: {
+                namespace: TelemetryNamespace.Lambda,
+                name: 'invoke_remote'
+            }
         })
 
         registerCommand({
             command: 'aws.configureLambda',
-            callback: configureLocalLambda
+            callback: configureLocalLambda,
+            telemetryName: {
+                namespace: TelemetryNamespace.Lambda,
+                name: 'configure'
```

Comment:
It seems very likely we will have the ability to configure remote lambdas in the future. This command is for configuring local lambdas. It is worth disambiguating at this time to avoid future confusion. Consider `configure_local` to parallel the `invoke_local` entries.

Reply:
This is still outstanding

Reply:
Do we have local Lambda config metrics on any of the other toolkits? Is this something we want to track?